pkgs/by-name/ru/ruff/package.nix

```diff
@@ -81,6 +81,8 @@ rustPlatform.buildRustPackage (finalAttrs: {
     tests = lib.optionalAttrs stdenv.hostPlatform.isLinux {
       nixos-test-driver-busybox = nixosTests.nixos-test-driver.busybox;
     };
+    # Updating ruff are merged into staging-next due to NixOS tests. Disabling r-ryantm update bot:
```

Comment:
have to ask about why staging-next ? 
specifically https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#staging


> The staging-next branch should then only receive changes that fix Hydra builds; for anything else, ask the [Staging room](https://matrix.to/#/#staging:nixos.org) first

Reply:
as per https://matrix.to/#/!UNVBThoJtlIiVwiDjU:nixos.org/$turTmiP_ee5CDBl_02LOR_asjbogNvCIbffp6FcRiZw?via=nixos.org&via=matrix.org&via=tchncs.de

if there is no active staging-next cycle, updates should go to staging

Reply:
I changed the comment.